### PR TITLE
[CARBONDATA-3111] Readme updated some error links had been fixed

### PR DESCRIPTION
**Pull request by RebieKong**
Readme updated
some error links that link to datamap had been link to correct file
 - [x] Any interfaces changed?
 No
 - [x] Any backward compatibility impacted?
 No
 - [x] Document update required?
 Yes， it is a document update     
 - [x] Testing done
No
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
No

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by xubo245** [comment]
@RebieKong Can you raise a JIRA for it? and then add JIRA number in title.  such as:[CARBONDATA-2999] support read schema from S3

and please keep :
`Any interfaces changed?

  Any backward compatibility impacted?

  Document update required?

  Testing done

  For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.
`

File: README.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ CarbonData is built using Apache Maven, to [build CarbonData](https://github.com
...
-* [CarbonData DataMap Management](https://github.com/apache/carbondata/blob/master/docs/datamap-management.md) 
- * [CarbonData BloomFilter DataMap](https://github.com/apache/carbondata/blob/master/docs/bloomfilter-datamap-guide.md) 
- * [CarbonData Lucene DataMap](https://github.com/apache/carbondata/blob/master/docs/lucene-datamap-guide.md) 
- * [CarbonData Pre-aggregate DataMap](https://github.com/apache/carbondata/blob/master/docs/preaggregate-datamap-guide.md) 
- * [CarbonData Timeseries DataMap](https://github.com/apache/carbondata/blob/master/docs/timeseries-datamap-guide.md) 
+* [CarbonData DataMap Management](https://github.com/apache/carbondata/blob/master/docs/datamap/datamap-management.md) 
+ * [CarbonData BloomFilter DataMap](https://github.com/apache/carbondata/blob/master/docs/datamap/bloomfilter-datamap-guide.md) 
+ * [CarbonData Lucene DataMap](https://github.com/apache/carbondata/blob/master/docs/datamap/lucene-datamap-guide.md) 
+ * [CarbonData Pre-aggregate DataMap](https://github.com/apache/carbondata/blob/master/docs/datamap/preaggregate-datamap-guide.md)
```
Comment (sraghunandan) [inline README.md]: Please remove the extra tab

Comment (RebieKong) [reply README.md]: ok~

**Comment by xubo245** [comment]
LGTM

**Comment by chenliang613** [comment]
add to whitelist

**Comment by chenliang613** [comment]
LGTM

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1481/

**Comment by sraghunandan** [comment]
LGTM

**Pull request closed without merging**